starter/src/main/java/com/starter/fullstack/dao/InventoryDAO.java

```diff
@@ -10,49 +11,46 @@
 import org.springframework.data.mongodb.core.index.IndexOperations;
 import org.springframework.util.Assert;
 
-/**
```

Comment:
So Hannah, you actually want to leave these comments in the /** form. These are called Javadoc comments, and what they do is provide important information to a developer that might want to implement this method in the future. Here's a good description of what they are used for: https://www.baeldung.com/javadoc